Makefile

```
@@ -1,3 +1,9 @@
+# end-to-end quantized model evaluation
+.PHONY: qbert
+qbert:
+	-bash scripts/build_qbert_env.sh
+	-bash scripts/eval_qbert.sh
```

환경 설정은 1회성이라 eval_qbert.sh 만 남겨두고 build_qbert_env.sh 는 제거하는 것이 좋을 것 같습니다.

```suggestion
.PHONY: qbert
qbert:
	-bash scripts/eval_qbert.sh
```

---

비슷한 코멘트여서 https://github.com/furiosa-ai/inference/pull/16#discussion_r1562085496 에서 아래와 같이 의견남겼습니다. 의견 부탁드리겠습니다.

> 제가 의도했던 것은 매 모델 평가 시 매번 환경 구성을 진행하는 것이 기본이고, 만약 환경 구성을 한 번하고 실험을 반복적으로 하고자하면 오히려 bash scripts/eval_qgpt-j.sh 를 반복해서 실행하자는 것이었습니다.
> 그렇게 생각한 이유는 한 번 conda env. activate 된 상황에서 다른 어떠한 이유로 환경이 바뀔 수가 있고, 사용자가 인지하지 못한 이유로 평가 결과가 바뀌는 것을 막기 위함이었습니다.
>

---

[review: make instruction simple](https://github.com/furiosa-ai/inference/pull/12/commits/c5fe20f6a85c8641502fec12cd3b41ee03fda9fc) 반영했습니다. 다시 한 번 생각해보니까 말씀해주신대로 구성하면 훨씬 깔끔한 것 같습니다.